oak_launcher_utils/src/launcher.rs

```diff
@@ -283,6 +283,8 @@ pub async fn launch(
 
             let mut line = String::new();
             while reader.read_line(&mut line).expect("couldn't read line") > 0 {
+                // remove the new line character
+                line.pop();
```

Comment:
Maybe https://doc.rust-lang.org/std/primitive.str.html#method.trim  instead?

Reply:
Good point. Though there is https://doc.rust-lang.org/std/primitive.str.html#method.trim_end which may work for us :) and it seems more semantically accurate to me. (but I was just curious to look around for string related functions to learn more about them -- no need to change this)